test/xUnit/csharp/test_Runspace.cs

```diff
@@ -66,7 +66,7 @@ public void TestRunspaceWithPowerShell()
         [Fact]
         public void TestRunspaceWithPowerShellAndInitialSessionState()
         {
-            InitialSessionState iss = InitialSessionState.CreateDefault2();
+            InitialSessionState iss = InitialSessionState.CreateDefault();
```

Comment:
Why was this changed?

Reply:
Two tests above use the `CreateDefault()`. I guess it was typo. If we need tests for `CreateDefault2()` I believe it should be added in new PR.

Reply:
I believe `CreateDefault2` is intentional as it creates an ISS without any modules loaded. Please change it back.

Reply:
Reverted with new comment.

Reply:
@adityapatwardhan that seems like a method name that probably needs revising, if we can, or adding an alias for in some fashion. Ilya is generally pretty familiar with the repo; if he got it confused, it's likely others will also.

Reply:
Changing the methodName would be a huge breaking change, but we could add a new method that calls this that is named better.  I would suggest filing an issue about this.